configs/microblaze_adi_defconfig

```diff
@@ -13,6 +13,7 @@ BR2_LINUX_KERNEL_CUSTOM_GIT=y
 BR2_LINUX_KERNEL_CUSTOM_REPO_URL="http://github.com/analogdevicesinc/linux"
 BR2_LINUX_KERNEL_CUSTOM_REPO_VERSION="master"
 BR2_LINUX_KERNEL_DEFCONFIG="adi_mb"
+BR2_PACKAGE_FRU_TOOLS=y
```

Comment:
unrelated to this;
should we [at some point] remove the `microblaze_adi_defconfig ` defconfig and leave the Vivado flavors only?
with those a user would know exactly which version of vivado to use

Reply:
sounds reasonable